cmd/container-disk-v3alpha/container-disk.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
If I recall the other program was written in C to reduce memory consumption.  What will memory consumption be now?

See https://github.com/kubevirt/kubevirt/pull/2844

Reply:
Good point. It may be better if we now import less packages, but probably still requires a c application

Reply:
Thanks for pointing this out. I didn't know that. I'll try to evaluate it

Reply:
We need a C binary unfortunately, the size of the binary is still large, ~ 24M